hack/testing/check-curator.sh

```diff
@@ -29,15 +40,20 @@ TEST_DIVIDER="------------------------------------------"
 # key=./key-fluentd
 # oc get secret logging-fluentd --template='{{.data.cert}}' | base64 -d > cert-fluentd
 # cert=./cert-fluentd
-fpod=`oc get pods | awk '/-deploy/ {next}; /-build/ {next}; /^logging-fluentd-.* Running / {print $1}'`
-
-# where "yesterday", "today", etc. are the dates in UTC in YYYY.mm.dd format
+fpod=`oc get pods -l component=fluentd | awk '/fluentd/ {print $1}'`
```

Comment:
well... it can change. it has in the past (beta timeframe). it's not technically part of the API, just a client implementation.

now you could also say "well if the template changes our test will break and we'll fix it." and that's true enough.

still, if it's not outlandishly complicated, i'd prefer to have the client give us exactly what we're looking for rather than try to extract things from its output.